src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let counter = 0;
+
+  return (b) => (a) => {
+    counter++;
+
+    const checkCounting = (counter < 3
+      || (counter >= 3 && counter % 2 !== 0))
```

Comment:
You don't need the additional check
```suggestion
    const returnValue = counter < 3 || counter % 2 !== 0
```